zh-cn-README.md

```diff
@@ -0,0 +1,44 @@
+# neofetch-win
+这是neofetch，但是为windows而写的。
+
+![PreviewImage](https://i.alexflipnote.dev/vfgQo1y.png)
+
+这是为了使[Neofetch](https://github.com/dylanaraps/neofetch)在Windows上的CMD上可用。如果您愿意捐款，请随时捐款。
+##### 译者提示：在Git Bash可能会出现错误，请尽量在Windows上的CMD上使用。
+
+## 安装需求
+- Python 3.6或更高版本。
+
+## 安装
+- 使用管理员权限打开CMD。
+- 输入以下命令：`pip install neofetch-win`。
+- 现在你可以在CMD输入`neofetch`来查看结果。
+
+### 可用的颜色
+黑色，红色，绿色，黄色，蓝色，洋红色，青色，白色。
```

Comment:
Could you add that the colours has to be written in English?

Reply:
okay

Reply:
please wait

Reply:
done!